core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -62,9 +62,11 @@
       </div>
       <div class="oppia-dashboard-language-container" ng-if="$ctrl.showLanguageSelector()">
         <span class="oppia-dashboard-language-container-label">Translate to</span>
-        <translation-language-selector [active-language-code]="$ctrl.languageCode"
-                                       (set-active-language-code)="$ctrl.onChangeLanguage($event)">
-        </translation-language-selector>
+        <button focus-on="selectLangDropDown" class="oppia-autofocus border-0 p-0">
```

Comment:
Can you add a video/gif showing what this looks like?

Reply:
Done, I have added the video in the Proff that changes are correct section.